cartridges/int_alma/cartridge/scripts/helpers/almaRefundHelper.js

```diff
@@ -46,9 +62,5 @@ exports.refundPaymentForOrder = function (order, amount) {
     if (httpResult.msg !== 'OK') {
         throw Error('Could not create refund on Alma side.');
     }
-
-    Transaction.wrap(function () {
-        order.custom.ALMA_Refunded = true; // eslint-disable-line no-param-reassign
-    });
 };
```

Comment:
It's here that we should affect `ALMA_Refunded_Amount`, this way, we know alma responded us with a 200 meaning the refund went well